libs/docs/platform/message-popover/message-popover-docs.component.html

```diff
@@ -0,0 +1,81 @@
+<fd-docs-section-title id="default" componentName="message-popover">
+    Message Popover with Simple forms
+</fd-docs-section-title>
+<description>
+    <p>Message popover component supports both template-driven and reactive forms.</p>
+    <p>
+        To define human-readable form item label text, developers need to use
```

Comment:
```suggestion
        To define human-readable error message text, developers need to use
```

Reply:
Hey @platon-rov. The original text is actually correct. With fdpMessagePopoverFormItem developers can define field name if they are not using form field component from platform lib (as example, they use core's controls).